pkg/pool-utils/contracts/controllers/ManagedPoolController.sol

```diff
@@ -77,13 +79,15 @@ contract ManagedPoolController is BasePoolController, IControlledManagedPool {
         pure
         returns (bytes32)
     {
-        bytes32 permissions = super.encodePermissions(baseRights);
+        bytes32 permissions = super
+            .encodePermissions(baseRights)
+            .insertBool(managedRights.canChangeWeights, _CHANGE_WEIGHTS_OFFSET)
+            .insertBool(managedRights.canDisableSwaps, _DISABLE_SWAPS_OFFSET)
+            .insertBool(managedRights.canSetMustAllowlistLPs, _MUST_ALLOWLIST_LPS_OFFSET);
 
         return
             permissions
-                .insertBool(managedRights.canChangeWeights, _CHANGE_WEIGHTS_OFFSET)
-                .insertBool(managedRights.canDisableSwaps, _DISABLE_SWAPS_OFFSET)
-                .insertBool(managedRights.canSetMustAllowlistLPs, _MUST_ALLOWLIST_LPS_OFFSET)
+                .insertBool(managedRights.canChangeMgmtSwapFee, _CHANGE_MGMT_SWAP_FEE_OFFSET)
                 .insertBool(managedRights.canChangeTokens, _CHANGE_TOKENS_OFFSET)
                 .insertBool(managedRights.canSetCircuitBreakers, _CIRCUIT_BREAKERS_OFFSET);
```

Comment:
I don't understand this change, why are some done on the first pass and some on the second? Stack too deep?

Reply:
> I don't understand this change, why are some done on the first pass and some on the second? Stack too deep?

Yes; that's always the problem when it looks weird. Adding one more was too many. I could add a comment.